luxonis_ml/data/datasets/luxonis_dataset.py

```diff
@@ -54,28 +54,18 @@ def __init__(
             C{dataset_name} nor C{dataset_id} are provided.
         """
 
-        if dataset_name is not None and not isinstance(dataset_name, str):
-            raise ValueError("`dataset_name` argument must be a string")
-
-        if dataset_name is None and dataset_id is None:
-            raise ValueError(
-                "Must provide either dataset_name or dataset_id when initializing LuxonisDataset"
-            )
```

Comment:
Is this validation handled elsewhere?

Reply:
No, this fell through the cracks. I'll add it back.